build.assets/tooling/cmd/build-apt-repos/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	supportedOSs := map[string][]string{
+		"debian": {
+			"stretch",
+			"buster",
+			"bullseye",
+			"bookwork",
+			"trixie",
```

Comment:
Can you add corresponding versions as comments here and below? E.g.

```go
"stretch",  // 9
"buster",   // 10
"bullseye", // 11
```

etc.

Reply:
Updated pending push